lowcar/devices/KoalaBear/PID.cpp

```diff
@@ -17,18 +17,20 @@ PID::PID() {
 float PID::compute(float curr_pos) {
     unsigned long curr_time = micros(); // get the current time
     float interval_secs = ((float)(curr_time - this->prev_time)) / 1000000.0; // compute time passed between loops, in seconds
-    float desired_pos = prev_desired_pos + (duty_cycle_to_tps(this->target_speed) * interval_secs); // compute the desired position at this time
+    float desired_pos = this->prev_desired_pos + (duty_cycle_to_tps(this->target_speed) * interval_secs); // compute the desired position at this time
```

Comment:
Not sure if it should say `curr_pos` instead of `this->prev_desired_pos` but I guess you can test that when you get the encoders.